node/src/components/consensus/era_supervisor.rs

```diff
@@ -144,6 +146,13 @@ where
         registry: &Registry,
         new_consensus: Box<ConsensusConstructor<I>>,
     ) -> Result<(Self, Effects<Event<I>>), Error> {
+        if current_era < protocol_config.last_activation_point {
+            panic!(
+                "Current era ({:?}) is before the last activation point ({:?}) - no eras would \
+                be instantiated!",
+                current_era, protocol_config.last_activation_point
+            );
```

Comment:
Should we return the `Fatal` effect instead?

Reply:
Possibly :+1:

Reply:
I'm not sure if we can, though - we'd have to construct a valid era supervisor, anyway, and return the `Fatal` effect along with it. Does that make sense, though?

Reply:
Right, I guess it doesn't.